src/legends/searchable-discrete-color-legend.js

```diff
@@ -59,24 +60,26 @@ function SearchableDiscreteColorLegend(props) {
     null;
   const filteredItems = searchFn(items, searchText);
   return (
-    <div className={`rv-search-wrapper ${className}`} style={{width, height}}>
-      <form className="rv-search-wrapper__form">
-        <input
-          type="search"
-          placeholder={searchPlaceholder}
-          className="rv-search-wrapper__form__input"
-          value={searchText}
-          onChange={onChange}/>
-      </form>
-      <div className="rv-search-wrapper__contents">
-        <DiscreteColorLegend
-          colors={colors}
-          items={filteredItems}
-          onItemClick={onItemClick}
-          orientation={orientation}
-        />
+    <LegendContainer>
```

Comment:
it'd probably be good to keep the custom class and identifying class on the top level tag in case the whole thing needs to be custom placed by external css.